tableau-databricks/connection-builder.js

```diff
@@ -29,15 +29,24 @@ limitations under the License.
 	var authenticationMode = attr[connectionHelper.attributeAuthentication];
```

Comment:
Could we add some usage logging for the connection string? The size of the string for instance, if it includes extra confs, etc. Not sure if the Simba log already gives such logs.  Logging from the connector should work with logging.Log("...").